college_flow_app/lib/features/list_subjects/widgets/searchbar.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:college_flow_app/config/design_system/data/spacing/spacing.dart';
+import 'package:college_flow_app/shared/widgets/flow_icon_button.dart';
+import 'package:college_flow_app/shared/widgets/gap.dart';
+import 'package:flutter/material.dart';
+
+import '../../../config/design_system/data/colors/colors.dart';
+import '../../../shared/widgets/flow_icon.dart';
+
+class SearchBar extends StatelessWidget {
+  const SearchBar({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      crossAxisAlignment: CrossAxisAlignment.center,
+      mainAxisAlignment: MainAxisAlignment.center,
+      mainAxisSize: MainAxisSize.max,
+      children: [
+        FlowIconButton(
+          icon: const FlowIcon.search(
+            color: colorSecondary,
+            size: spacingXS,
+          ),
+          onTap: () {},
+        ),
+        const VSpacer.quarck(),
+        SizedBox(
+          width: 280,
+          height: 32,
+          child: TextFormField(
+            decoration: InputDecoration(
+              enabledBorder: OutlineInputBorder(
+                borderSide: const BorderSide(color: colorSecondary, width: 1),
+                borderRadius: BorderRadius.circular(50.0),
+              ),
+              hintText: 'Digite as disciplinas que deseja buscar',
+              hintStyle: Theme.of(context).textTheme.bodySmall?.copyWith(
+                    color: colorDarkWhite,
+                  ),
+              isDense: true,
+              contentPadding: const EdgeInsets.all(8.0),
+            ),
+          ),
+        ),
```

Comment:
Remove Sized Box. You can use widgets like expanded or flexible to make the textField use all the space needed.
Also, why not use the FlowTextField you can add new props and use our design system.

Reply:
the textfield in Searchbar is not the same we are using in the normal textfields